index.js

```diff
@@ -0,0 +1,77 @@
+import { displayBooksControl, removeBook, addBook } from './modules/display.js';
+import {
+  displayBooks, displayForm, contact, displayContact,
+  showBooksOnly, showOnlyContact, showOnlyForm,
+} from './modules/dynamic.js';
+
+import { DateTime } from './modules/luxon.js';
+
+const bookItems = document.querySelector('#book-list');
+const awesomeHeading = document.querySelector('#awesome');
+
+class Book {
+  constructor(title, author) {
+    this.title = title;
+    this.author = author;
+  }
+}
+
+// display  all the books
```

Comment:
- [ ] [OPTIONAL] I noticed that you almost comment on every part of your code. Please try to keep comments as short as you can with minimal context. If the reason you add a comment is that the code is hacky or complicated, then make it readable by refactoring it — not by adding another confusing layer. Choose better names, break functions to do one thing, and use abstractions :+1: